src/main/java/com/googlecode/download/maven/plugin/internal/WGetMojo.java

```diff
@@ -276,6 +277,19 @@ public class WGetMojo extends AbstractMojo {
     @Parameter(property = "preemptiveAuth", defaultValue = "false")
     private boolean preemptiveAuth;
 
+    
+    private void adjustOutputDirectory() throws MojoExecutionException {
```

Comment:
@culmat I'd appreciate javadoc for the method

Reply:
Here we go :-)